packages/react-components/react-charts-preview/src/components/LineChart/LineChartRTL.test.tsx

```diff
@@ -537,7 +537,7 @@ describe('Line chart - Subcomponent legend', () => {
     },
     container => {
       const legends = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'button');
-      expect(legends).toHaveLength(6);
+      expect(legends).toHaveLength(5);
```

Comment:
Was there any change in number of legends?

Reply:
Yes, In V8 it is showing 6 but in V9 the count is 5.

Reply:
Why 5 in V9? The legend count should not be same?

Reply:
Actually, by mistakenly I mentioned the "V8 count is 6" the V8 count is 5 only. Inside V9 the count was 6 which I now changed it 5.